inference-engine/tests/functional/plugin/myriad/single_layer_tests/gather.cpp

```diff
@@ -0,0 +1,120 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "single_layer_tests/gather.hpp"
+#include <vpu/private_plugin_config.hpp>
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+
+using GatherParams = std::tuple<
+    std::vector<size_t>,                 // Indices shape
+    std::pair<std::vector<size_t>, int>, // Input shapes and axis
+    InferenceEngine::Precision           // Network precision
+>;
+
+const std::vector<std::vector<std::size_t>> indicesShapes = {
+    {},
+    {5},
+    {10, 5},
+    {1, 128, 1},
+    {15, 4, 20, 5},
+};
+
+const std::vector<std::pair<std::vector<std::size_t>, int>> inputShapes = {
+    {{6, 12, 10, 24}, -4},
+    {{6, 12, 10, 24}, -3},
+
+    {{3052, 768},     -2},
+    {{6, 12, 10, 24}, -2},
+
+    {{10},            -1},
+    {{3052, 768},     -1},
+    {{6, 12, 10, 24}, -1},
+
+    {{10},             0},
+    {{3052, 768},      0},
+    {{6, 12, 10, 24},  0},
+
+    {{3052, 768},      1},
+    {{6, 12, 10, 24},  1},
+
+    {{6, 12, 10, 24},  2},
+    {{6, 12, 10, 24},  3},
+};
+
+const std::vector<InferenceEngine::Precision> networkPrecisions = {
```

Comment:
Are these precisions ok for testing? Or should we leave only one?

Reply:
What does it mean `ok`? Tests are passing as you can see from CI.